pvHelpers/email/email_helpers.py

```diff
@@ -15,7 +15,8 @@ class PROTOCOL_VERSION(object):
     V3 = 3
     V4 = 4
     V5 = 5
-    Latest = 4
+    V6 = 6
```

Comment:
This becomes a bit ambiguous.
It could represent either a private metadata's protocol version or that of an email.

Should we split them? I am open to suggestions.